amid/ribfrac/dataset.py

```diff
@@ -0,0 +1,104 @@
+import os
+
+import nibabel
+import numpy as np
+from connectome import Output, Source, meta
+from connectome.interface.nodes import Silent
+from dpipe.io import load
+
+from ..internals import checksum, licenses, register
+from ..utils import deprecate
+
+
+@register(
+    body_region='Chest',
+    license=licenses.CC_BYNC_40,
+    link='https://ribfrac.grand-challenge.org',
+    modality='CT',
+    raw_data_size='77.8 G',
+    task='Segmentation',
+)
+@checksum('ribfrac')
+class RibFrac(Source):
+    """
+    RibFrac dataset is a benchmark for developping algorithms on rib fracture detection,
+    segmentation and classification. We hope this large-scale dataset could facilitate
+    both clinical research for automatic rib fracture detection and diagnoses,
+    and engineering research for 3D detection, segmentation and classification.
+
+
+    Parameters
+    ----------
+    root : str, Path, optional
+        path to the folder containing the raw downloaded archives.
+        If not provided, the cache is assumed to be already populated.
+
+
+    Notes
+    -----
+    Data downloaded from here:
+    https://doi.org/10.5281/zenodo.3893507 -- train Part1 (300 images)
+    https://doi.org/10.5281/zenodo.3893497 -- train Part2 (120 images)
+    https://doi.org/10.5281/zenodo.3893495 -- val (80 images)
+    https://zenodo.org/record/3993380 -- test (160 images without annotation)
+
+
+
+    References
+    ----------
+    Jiancheng Yang, Liang Jin, Bingbing Ni, & Ming Li. (2020).
+    RibFrac Dataset: A Benchmark for Rib Fracture Detection,
+    Segmentation and Classification
+    """
+
+    _root: str = None
```

Comment:
use a _base(root: Silent) parameter to make some basic checks. look at bimcv for an example